tests/python/driver/tvmc/test_command_line.py

```diff
@@ -155,3 +162,39 @@ def test_tvmc_tune_file_check(capsys, invalid_input):
     )
     on_assert_error = f"'tvmc tune' failed to check invalid FILE: {invalid_input}"
     assert captured.err == expected_err, on_assert_error
+
+
+@mock.patch("tvm.relay.build")
+@mock.patch("tvm.driver.tvmc.load")
+@mock.patch("tvm.transform.PassContext")
+@mock.patch("tvm.driver.tvmc.model.TVMCPackage.__init__", return_value=None)
+@mock.patch("tvm.driver.tvmc.composite_target.get_codegen_by_target")
+def test_compile_check_workspace_pools(mock_ct, mock_pkg, mock_pc, mock_fe, mock_relay):
```

Comment:
This would be better suited to `test_compiler.py` and another test here which calls `_main` with the workspace parameters in argument form.

Reply:
ack to both moving the test and a different test for `test_command_line.py`